tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -135,3 +135,19 @@ spec:
           # value expected by Tekton Pipelines. NOTE: If params.SSL_CERT_DIR is empty, the value
           # will contain a trailing ":" - this is ok.
           value: "/tekton-custom-certs:/etc/ssl/certs:/etc/pki/tls/certs:/system/etc/security/cacerts:$(params.SSL_CERT_DIR)"
+    - name: display
+      image: quay.io/hacbs-contract/ec-cli:snapshot
+      command: [jq]
```

Comment:
We may have to install this in the ec-cli image?

Reply:
[Yeah](https://github.com/enterprise-contract/ec-cli/actions/runs/4558491080/jobs/8041441182?pr=572#step:5:552):
```
Error executing command: exec: "jq": executable file not found in $PATH
```

Reply:
I don't know if the top-level Dockerfile is the right place to install `jq`, but that's where I put it. I can move if needed.